drinfomon.lic

```diff
@@ -1439,7 +1438,7 @@ while line = script.gets
         .each_key do |town|
           copper = convert2copper(Regexp.last_match(1).to_i, Regexp.last_match(2))
           CharSettings['bank_accounts'][town] = CharSettings['bank_accounts'][town].to_i + copper
-          break
+          # break
```

Comment:
Why was this commented out?

Reply:
Rubocop suggested this loop would only ever run once in any case
```
drinfomon.lic:1437:7: W: Lint/UnreachableLoop: This loop will have at most one iteration.
      bank_titles ...
      ^^^^^^^^^^^
drinfomon.lic:1449:7: W: Lint/UnreachableLoop: This loop will have at most one iteration.
      bank_titles ...
      ^^^^^^^^^^^
```

Reply:
We really should only ever have one item match the `select` a few lines up. The break just makes that explicit.

I suggest changing `select` to `find` (and deleting the `break`), which will also have a meaningless performance improvement since Ruby doesn't need to traverse the entire list.